lib/vagrant-service-manager/command.rb

```diff
@@ -17,6 +17,8 @@ module ServiceManager
       'docker' => Docker, 'openshift' => OpenShift,
       'kubernetes' => Kubernetes
     }
+    # Download binary at current folder
+    BIN_FOLDER = "#{ENV['HOME']}/.vagrant.d/data/service-manager/bin"
```

Comment:
So are you going to  fix it? It is either part of the @env instance you are getting passed - https://github.com/vagrant-landrush/landrush/blob/master/lib/landrush/command.rb#L11 or you need to use the home_path - https://github.com/vagrant-landrush/landrush/blob/master/lib/landrush/action/setup.rb#L9. You need to test.